cmd/cosign/cli/attest/attest.go

```diff
@@ -67,6 +67,12 @@ func AttestCmd(ctx context.Context, ko sign.KeyOpts, regOpts options.RegistryOpt
 		return errors.Wrap(err, "parsing reference")
 	}
 
+	if timeout != time.Duration(0) {
```

Comment:
This reads a lot better
```suggestion
	if timeout != 0 {
```

Reply:
done